R/create_target_metadata_item.R

```diff
@@ -11,13 +11,14 @@
 #' @param target_name character string. A longer human readable target description
 #' that could be used, for example, as a visualisation axis label.
 #' @param target_units character string. Unit of observation of the target.
-#' @param target_keys named list or `NULL`. Should be `NULL`, in the case
-#' where the target is not specified as a task_id but is specified solely through
-#' the `target_id` argument. Otherwise, should be a named list of one or more
-#' character strings. The name of each element should match a task_id variable
-#' within the same model_tasks object. Each element should be of length 1.
-#' Each value, or the combination of values if multiple keys are specified,
-#' define a single target value.
+#' @param target_keys named list or `NULL`. The `target_keys` value defines a
+#' single target.
+#' Should be `NULL`, in the case where the target is not specified as a task_id
+#' but is specified solely through the `target_id` argument.
+#' Otherwise, should be a named list containing a single character string element.
+#' The name of the element should match a `task_id`
+#' variable name within the same `model_tasks` object and the value should match
+#' a single value of that variable.
```

Comment:
Adding this link as it provides a useful diagram for this relationship. 

```suggestion
#' a single value of that variable as described in [target metadata section of the official hubverse documentation](https://hubverse.io/en/latest/user-guide/tasks.html#target-metadata).
```